modules/ROOT/pages/getting-started.adoc

```diff
@@ -7,63 +7,99 @@ xref:glossary:index.adoc[glossary].
-its latest built commit, initial build configuration parameters (i.e. Containerfile path), and any relationships to
+its latest built commit, initial build configuration parameters, and any relationships to
 other Components. The CR also contains a reference to its single owning Application.
-Component names must be unique in a namespace, even when components are used in different application.
-
-=== Pipelines as Code (PAC)
-https://pipelinesascode.com/[Pipelines as Code] is a project that provides opinionated tooling to define a
-Tekton-based CI/CD pipeline. It enables the use of a Tekton PipelineDefinition within a git repository and for
-that pipeline to be triggered upon new commits/PR to the repository. In Konflux, these pipelines are pushed to
-the git repositories in the `.tekton` directory.
+Component names are unique in a namespace, even when components are used in different application.
 
 === Build pipeline
-Upon a new push or pull request (merge request in GitLab) event, the pipeline defined in the repository will be
+When you create a Component in {ProductName}, a build pipeline is pushed to the git repository in the `.tekton/`
+directory and a webhook is installed.
+
+Upon a new push or pull request (merge request in GitLab) event, the pipeline defined in the repository is
 run. This pipeline describes the process to build and test a specific artifact. The build process includes Tekton
-Tasks like cloning the git repository, prefetching dependencies, building the OCI artifact and source SBOM, and
-generating the source container. The test process includes Tekton Tasks like running Snyk scans, checking for
+Tasks such as cloning the git repository, prefetching dependencies, building the OCI artifact and source SBOM, and
+generating the source container. The test process includes Tekton Tasks such as running Snyk scans, checking for
 CVEs with clair-in-ci, and running an antivirus scan on the artifact.
 
+{ProductName} inherits the pattern of defining Tekton build pipelines in git
+from its use of https://pipelinesascode.com/[Pipelines as Code (PaC)]. PaC
+enables the use of a Tekton PipelineDefinition within a git repository and for
+that pipeline to be triggered upon new commits and PRs to the repository.
+
+Artifacts that you build in your tenant namespace (from git pull request, merge request or push events) are pushed to an OCI registry
+along with their supporting metadata (including xref:metadata:index.adoc[SLSA provenance attestations and SBOMs]).
+
 === Application
-An Application CR owns multiple Components. You can think of it as the closest object in {ProductName} that
-models an ongoing supported product version. If for some reason you decide you need to, you can decompose your
-product version further and model different sub-products each as their own Application.
+An Application CR owns multiple Components. You can think of it as the object in {ProductName} that
+models an ongoing supported product version.
```

Comment:
This feels like a Red Hat ism. Can/should we describe it more generically?

Reply:
I agree, it's a Red Hat ism. Let's just remove it!